app/services/utilities/new_payment_period_analyser.rb

```diff
@@ -0,0 +1,16 @@
+module Utilities
+  class NewPaymentPeriodAnalyser
```

Comment:
Does this need to be prefixed *New*?
Just thinking bout future devs, I'd be wondering if there is an _original_ PaymentPeriodAnalyser

Reply:
yep - I'll change it.